.github/workflows/build-docker.yml

```diff
@@ -295,14 +317,39 @@ jobs:
 
       # Adapted from https://docs.docker.com/build/ci/github-actions/multi-platform/
       - name: Create manifest list and push
+        id: manifest-push
         working-directory: /tmp/digests
         # The readarray part is used to make sure the quoting and special characters are preserved on expansion (e.g. spaces)
         # The jq command expands the docker/metadata json "tags" array entry to `-t tag1 -t tag2 ...` for each tag in the array
         # The printf will expand the base image with the `<UV_BASE_IMG>@sha256:<sha256> ...` for each sha256 in the directory
         # The final command becomes `docker buildx imagetools create -t tag1 -t tag2 ... <UV_BASE_IMG>@sha256:<sha256_1> <UV_BASE_IMG>@sha256:<sha256_2> ...`
+        # The digest of the new manifest is then shared as the 'digest' output
         run: |
           readarray -t lines <<< "$DOCKER_METADATA_OUTPUT_ANNOTATIONS"; annotations=(); for line in "${lines[@]}"; do annotations+=(--annotation "$line"); done
           docker buildx imagetools create \
             "${annotations[@]}" \
             $(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
             $(printf '${{ env.UV_BASE_IMG }}@sha256:%s ' *)
+          # unfortunately "docker buildx imagetools create" does not yet have a
+          # clean way of sharing the digest of the manifest it creates, see
+          # https://github.com/docker/buildx/issues/2407. Using a separate command to
+          # retrieve it _now_ is better than doing it later though, as it is highly
+          # unlikely that the digest changes in the same job with the same local docker setup.
```

Comment:
What does this last sentence mean? I don't quite follow.

Reply:
Might be useful to explain, in a separate step, what the command does (e.g. outputs) and clarify that the issue is that `docker buildx imagetools create` doesn't return a list of updated digests.

Reply:
I've expanded the comment, explaining why it is being done in the same step, what the digest is needed for and what each command does.